packages/block-editor/src/hooks/layout.js

```diff
@@ -465,27 +465,6 @@ function BlockWithChildLayoutStyles( { block: BlockListBlock, props } ) {
  */
 export const withChildLayoutStyles = createHigherOrderComponent(
 	( BlockListBlock ) => ( props ) => {
-		const layout = props.attributes.style?.layout ?? {};
-		const { selfStretch, flexSize } = layout;
-		const hasChildLayout = selfStretch || flexSize;
-
-		const shouldRenderChildLayoutStyles = useSelect(
-			( select ) => {
-				// The callback returns early to avoid block editor subscription.
-				if ( ! hasChildLayout ) {
-					return false;
-				}
-
-				return ! select( blockEditorStore ).getSettings()
-					.disableLayoutStyles;
```

Comment:
Will this removal mean the styles and classnames get generated and output if `disableLayoutStyles` is `true`?

Reply:
Oh, of course, well spotted! Let me see if this still works if we keep it.

Reply:
Ok I think I've fixed it now 😅